internal/tuf/tuf.go

```diff
@@ -338,3 +354,42 @@ type OtherRepository interface {
 	// sign the other repository's initial gittuf root of trust metadata.
 	GetInitialRootPrincipals() []Principal
 }
+
+// HookStage encodes the Git stage at which a hook is to run
+type HookStage uint
+
+const (
+	PreCommit HookStage = iota
```

Comment:
```suggestion
	HookStagePreCommit HookStage = iota
```

nit: helpful to avoid collisions imo